### PR TITLE
Logout Functionality + Fix Login Code For Deployment

### PR DESCRIPTION
**Pull request by lyskevin**

### CODE REVIEW

File: webproject/src/main/java/com/google/sps/servlets/LandingServlet.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+     * @param response An HttpServletResponse object that contains the response the servlet sends to the client.
+     * @throws IOException If an input or output error is detected when the servlet handles the GET request.
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html");
+        response.getWriter().println(HtmlParser.parseHtmlFromFile("landing.html"));
+    }
+}
+
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/servlets/LandingServlet.java]: duplicate newline

File: webproject/src/main/java/com/google/sps/servlets/ListingsServlet.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+     * @param response An HttpServletResponse object that contains the response the servlet sends to the client.
+     * @throws IOException If an input or output error is detected when the servlet handles the GET request.
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html");
+        response.getWriter().println(HtmlParser.parseHtmlFromFile("listings.html"));
+    }
+}
+
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/servlets/ListingsServlet.java]: duplicate newline

File: webproject/src/main/java/com/google/sps/servlets/LogoutServlet.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+     * @param request An HttpServletRequest object that contains the request the client has made of the servlet.
+     * @param response An HttpServletResponse object that contains the response the servlet sends to the client.
+     * @throws IOException If an input or output error is detected when the servlet handles the server's request.
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.getWriter().print(authenticationHandler.getLogoutUrl("/"));
+    }
+}
+
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/servlets/LogoutServlet.java]: duplicate newline

File: webproject/src/main/java/com/google/sps/util/HtmlParser.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        while ((lineFromFile = bufferedReader.readLine()) != null) {
+            stringBuilder.append(lineFromFile);
+        }
+
+        bufferedReader.close();
+
+        return stringBuilder.toString();
+    }
+}
+
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/util/HtmlParser.java]: duplicate newline

**Review by kevinluvian — APPROVED** [review]

**Review by arjavibahety — APPROVED** [review]

**Pull request merged**